src/gui/org/deidentifier/arx/gui/view/impl/define/ViewAttributeTransformation.java

```diff
@@ -545,6 +545,15 @@ public void actionUpdateAttributeTypes(AttributeType typeNew) {
                     AttributeType type = model.getInputDefinition().getAttributeType(attribute);
                     // Type changed
                     if (type != typeNew) {
+                        // Change selection in combo
+                        if (attribute.equals(model.getSelectedAttribute())) {
```

Comment:
Wouldn't it be more intuitive to test for equality with "this.attribute"? AFAICT "this.attribute" and "model.getSelectedAttribute()" should always be equal, however, testing against the former seems to be more intuitive as we are manipulating the view represented by the current object...

Reply:
It probably is more intuitive, so I changed it. Please review.